sopel/modules/help.py

```diff
@@ -20,9 +20,8 @@
 
 import requests
 
-from sopel.config.types import ChoiceAttribute, StaticSection, ValidatedAttribute
-from sopel.module import commands, example, priority, rule
-from sopel.tools import SopelMemory
+from sopel import plugin, tools
+from sopel.config import types
```

Comment:
I prefer `from package import namespace` whenever possible, because it keeps the "namespace" behavior of Python's module.

But yeah, I'm going to fix this inconsistency!